### ISSUE TITLE
Training my model isn't working

### ISSUE THREAD

**Issue by Poco1232** [JarodMica/ai-voice-cloning#16]
When trying to train my AI voice, I finish the first two steps of the process. When I get to the Run Training portion it freezes and never gets to the graph part.

﻿dist: False
24-01-19 12:00:52.232 - INFO: Random seed: 7040
24-01-19 12:00:54.259 - INFO: Number of training data elements: 36, iters: 1
24-01-19 12:00:54.259 - INFO: Total epochs needed: 500 for iters 500
C:\Users\sirpo\Downloads\ai-voice-cloning\runtime\lib\site-packages\transformers\configuration_utils.py:363: UserWarning: Passing `gradient_checkpointing` to a config initialization is deprecated and will be removed in v5 Transformers. Using `model.gradient_checkpointing_enable() instead, or if you are using the `Trainer` API, pass `gradient_checkpointing-True` in your `TrainingArguments`.

After this text is shown nothing else happens. I tried reconnecting or stopping the training but that ends up breaking the site making everything infinitely load and forcing me to close the tab and command prompt and start over.

**Comment by JarodMica** [comment]
Were you able to make progress on your issue here?  Sometimes, the validation.txt file is filled with lines that should go into train.txt.  You can try swapping the names and seeing if this resolved it.  I don't see any errors in the above stack so it doesn't look like anything is wrong from the messages unfortunately. 

Feel free to open this back up if you are still having trouble.
